Skip pkgconfig lookup for OpenSSL on FreeBSD.
The values are passed via CMAKE_ARGS in the port Makefile.

--- CMakeLists.txt.orig	2017-02-12 14:05:09 UTC
+++ CMakeLists.txt
@@ -10,7 +10,9 @@ SET(FISH_VERSION ${VERSION_MAJOR}.${VERS
 FIND_PACKAGE(PkgConfig REQUIRED)
 
 PKG_CHECK_MODULES(GLIB_PKG REQUIRED glib-2.0)
-PKG_CHECK_MODULES(OPENSSL REQUIRED openssl)
+if(NOT CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
+	PKG_CHECK_MODULES(OPENSSL REQUIRED openssl)
+endif()
 
 if(GLIB_PKG_FOUND)
 	INCLUDE_DIRECTORIES(${GLIB_PKG_INCLUDE_DIRS})
